site/landing/layouts/index.html

```diff
@@ -49,17 +58,18 @@ <h1 class="logo">
         <li class="main-nav__item">
           <a href="{{ .Site.Params.one_year_url }}">Blog</a>
         </li>
-        <li class="main-nav__item user-toggle">
-          <button class="[ toggle-button ] [ js-mode-toggle ]" aria-label="Switch light and dark theme">
-            <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
-              <path fill="currentColor"
-                d="M12 2a10 10 0 01.3 20H12a10 10 0 110-20zm0 2a8 8 0 100 16v-3a5 5 0 010-10V4zm0 4v8a4 4 0 100-8z"
-                fill-rule="evenodd" />
-            </svg>
-          </button>
-        </li>
       </ul>
     </nav>
+
+    <div class="user-toggle">
+      <button class="[ toggle-button ] [ js-mode-toggle ]" aria-label="Switch light and dark theme">
+        <svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+          <path fill="currentColor"
+            d="M12 2a10 10 0 01.3 20H12a10 10 0 110-20zm0 2a8 8 0 100 16v-3a5 5 0 010-10V4zm0 4v8a4 4 0 100-8z"
+            fill-rule="evenodd" />
+        </svg>
+      </button>
+    </div>
```

Comment:
Would it be nicer to have this inside the hamburger menu?

Reply:
I thought it looked a bit odd under all the links but happy to move it in if you think it fits?

Reply:
(odd meaning out of place, i.e it's a control button, not a navigation link)

Reply:
Yeah, it might look a bit weird under the links, but at the moment we are drawing more attention to it by leaving it outside of the menu.